Enable QAT for static llama definition

navsud opened the pull request:
Summary: The model needed small modifications to be able to run QAT on GPUs.

Differential Revision: D79841467

lucylq reviewed (approved):
LGTM on the executorch changes. Could you run lintrunner?

navsud committed:
Enable QAT for static llama definition (#13285)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/13285

The model needed small modifications to be able to run QAT on GPUs.

Reviewed By: YIWENX14

Differential Revision: D79841467
